services/maintenance-page/docker-compose.yml.j2

```diff
@@ -26,7 +26,7 @@ services:
         - traefik.docker.network=${PUBLIC_NETWORK}
         - traefik.http.routers.{{"maintenance_page_web_" + j2item.replace('@','').replace(' ','').replace('.','').replace('-','').replace('\'','') + "_html"}}.priority={{MAINTENANCE_PAGES_TRAEFIK_PRIORITY}}
         - traefik.http.routers.{{"maintenance_page_web_" + j2item.replace('@','').replace(' ','').replace('.','').replace('-','').replace('\'','') + "_html"}}.rule={{  "Host(`" + j2item + "`)" }}
-        - traefik.http.routers.{{"maintenance_page_web_" + j2item.replace('@','').replace(' ','').replace('.','').replace('-','').replace('\'','') + "_html"}}.rule=(Host(`{{j2item}}`) && PathPrefix(`/`)) || (HostRegexp(`services.{{j2item}}`,`{subhost:[a-zA-Z0-9-]+}.services.{{j2item}}`) && PathPrefix(`/`)) || (HostRegexp(`services.testing.{{j2item}}`,`{subhost:[a-zA-Z0-9-]+}.services.testing.{{j2item}}`) && PathPrefix(`/`))
+        - traefik.http.routers.{{"maintenance_page_web_" + j2item.replace('@','').replace(' ','').replace('.','').replace('-','').replace('\'','') + "_html"}}.rule=(Host(`{{j2item}}`) && PathPrefix(`/`)) || (HostRegexp(`services.{{j2item}}`,`{subhost:[a-zA-Z0-9-]+}.services.{{j2item}}`) && PathPrefix(`/`))
```

Comment:
do you have some sort of test for theses routes?

Reply:
no, neither for the j2 nor for the routes. if you have smart suggestions for how to test this let me know :) thanks!

Reply:
I guess an obvious test is to run replacement against some samples that failed in the past
We do that with regex all the time for instance in https://github.com/ITISFoundation/osparc-simcore/blob/master/packages/models-library/tests/test_basic_regex.py


If you want to go further, another possibility is to use hypothesis (property-based testing) to automatically produce data and find edge cases.